src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,16 +1,11 @@
 package mate.academy.service;
 
 public class AuthenticationService {
-    /**
-     * Imagine that some user wants to login to your site.
-     * You should check if user credentials (login and password) are valid or not.
-     * All users are stored in <code>UserService</code> class.
-     * @param email - user's email
-     * @param password - user's password
-     * @return true if user by email exists and passed password is equal to user's password.
-     * Return false in any other cases.
-     */
+
     public boolean login(String email, String password) {
-        return false;
+        if (new UserService().findByEmail(email) == null) {
+            return false;
+        }
+        return new UserService().findByEmail(email).getPassword().equals(password);
```

Comment:
Can you create only one instance of UserService? Can you call the method `findByEmail(email)` only once?